src/test/resources/features/notUiFeatures/operatorhubTest.feature

```diff
@@ -3,13 +3,43 @@
 Feature: OperatorHub installation test
 
   Scenario: test operatorhub installation
+    Given clean openshift after operatorhub test
     When delete running instance of apicurito
+    Given setup operatorhub on cluster
     And deploy operator from operatorhub
     Then check that apicurito operator is deployed and in running state
 
     When deploy "first" custom resource
     Then check that apicurito "image" is "registry.redhat.io/fuse7/fuse-apicurito"
-    And check that apicurito "operator" is "registry.redhat.io/fuse7/fuse-apicurito-rhel7-operator"
+    And check that apicurito "operator" is "registry.redhat.io/fuse7/fuse-apicurito-rhel8-operator"
+    And check that "GENERATOR" has "2" pods
+    And check that "SERVICE" has "2" pods
+    And check that name and image of operator in operatorhub are correct
+
+    #Run simple smoke test
+    When delete API "tmp/download/openapi-spec.json"
+    And delete API "tmp/download/openapi-spec.yaml"
+    And log into apicurito
+    And import API "src/test/resources/preparedAPIs/basic.json"
+    And save API as "json" and close editor
+    And import API "tmp/download/openapi-spec.json"
+    Then check that API name is "testAPI"
+
+    When clean openshift after operatorhub test
+    And delete running instance of apicurito
+    Then reinstall apicurito
+
+  @operatorhub-upgrade
```

Comment:
I would change the test. This is not testing upgrade in the way we want to.  We want to install Apicurito from operatorhub (operator and UI+generator) (Basically first test^^) After that we want to upgrade operator and check that all resources are updated operator UI and generator all pod are from new images.